ToDo/src/main/java/com/todo/demo/Note/Note.java

```diff
@@ -19,11 +20,50 @@
     private String description;
     private Priority priority;
 
-    public Note(NoteThrift note) {
-        this.id = note.getId();
-        this.name = note.getName();
-        this.description = note.getDescription();
-        this.priority = Priority.LOW;
+    public Note() {
+    }
+
+    public Note toNote(NoteThrift noteThrift) {
+        Note note = new Note();
```

Comment:
dont use empty constructor, check @AllArgsConstructor and @Builder in lombok project

Reply:
Removed